front/src/common/api/osrdEditoastApi.ts

```diff
@@ -1895,68 +1883,84 @@ export type RollingStockUsage = {
     train_schedule_id: number;
   };
 };
-export type SearchTrackResult = {
+export type SearchResultItemTrack = {
   infra_id: number;
   line_code: number;
   line_name: string;
 };
-export type SearchOperationalPointResult = {
+export type GeoJsonPointValue = number[];
+export type GeoJsonPoint = {
+  coordinates: GeoJsonPointValue;
+  type: 'Point';
+};
+export type SearchResultItemOperationalPoint = {
   ch: string;
-  geographic: Point;
-  infra_id?: string;
+  geographic: GeoJsonPoint;
+  infra_id: number;
   name: string;
   obj_id: string;
-  schematic: Point;
+  schematic: GeoJsonPoint;
   track_sections: {
     position: number;
     track: string;
   }[];
   trigram: string;
-  uic?: number;
+  uic: number;
 };
-export type SearchSignalResult = {
-  geographic: Point;
+export type SearchResultItemSignal = {
+  geographic: GeoJsonPoint;
   infra_id: number;
   label: string;
-  line_code: string;
+  line_code: number;
   line_name: string;
-  schematic: Point;
+  schematic: GeoJsonPoint;
   settings: string[];
   signaling_systems: string[];
   sprite?: string | null;
   sprite_signaling_system?: string | null;
 };
-export type SearchStudyResult = {
-  description?: string;
+export type SearchResultItemProject = {
+  description: string;
   id: number;
+  image: number;
   last_modification: string;
   name: string;
-  project_id: number;
-  scenarios_count?: number;
-  tags?: string[];
+  studies_count: number;
+  tags: string[];
 };
-export type SearchProjectResult = {
+export type SearchResultItemStudy = {
   description: string;
   id: number;
-  image?: number;
   last_modification: string;
   name: string;
-  studies_count?: number;
+  project_id: number;
+  scenarios_count: number;
   tags: string[];
 };
-export type SearchScenarioResult = {
-  description?: string;
-  electrical_profile_set_id?: number;
+export type SearchResultItemScenario = {
+  description: string;
+  electrical_profile_set_id: number;
   id: number;
   infra_id: number;
-  infra_name?: string;
+  infra_name: string;
   last_modification: string;
   name: string;
   study_id: number;
-  tags?: string[];
-  trains_count?: number;
+  tags: string[];
+  trains_count: number;
+};
+export type SearchResultItem =
+  | SearchResultItemTrack
+  | SearchResultItemOperationalPoint
+  | SearchResultItemSignal
+  | SearchResultItemProject
+  | SearchResultItemStudy
+  | SearchResultItemScenario;
+export type SearchPayload = {
+  dry?: boolean;
+  object: string;
+  query: any;
```

Comment:
I don't like the any here :) can  you change that ?
```suggestion
  query: any;
```